File: Funções/Função Calc/Calc.py
input(("Bem vindo a calculadora do PP.\nNós diga seu nome: "))
def Soma(x,y):
    return (x+y)

def Subtração(x,y):
    return (x - y)

def Multiplicação(x,y):
    return (x*y)

def Divisão(x,y):
    if y == 0:
        return "Erro, divisão por 0!"
    else:
        return (x/y)

x = int(input("Digite um número: "))
y = int(input("Digite outro número:"))

escolha =int(input("Escolha: 1-adicão 2-subtração 3-Multiplicação 4-Divisão\n"))

if escolha == 1:
    print(Soma(x,y))   
elif escolha == 2:
    print(Subtração(x,y))
elif escolha == 3:
    print(Multiplicação(x,y))
elif escolha == 4:
    print(Divisão(x,y))

else:
    print("Erro, faça novamente!")

File: Funções/Função Calc/Calc.java
class Calc {

    public static void main(String[] args){
        int escolha = (int)(Math.random()*4+1);
        System.out.println("escolha: \n 1 - Soma \n 2 - Subtração \n 3 - Multiplicação \n 4 - Divisão ");
        System.out.println("Escolheu:\n" + escolha);

        int primeiroNumero = (int) (Math.random()*10+1);
        System.out.println("Primeiro Número:\n" + primeiroNumero);
        
        int segundoNumero = (int) (Math.random()*10+1);
        System.out.println("Segundo Número:\n" + y);

        if(escolha == 1){
        System.out.println("Resultado\n" + (primeiroNumero+segundoNumero));
        }else if(escolha == 2){
        System.out.println("Resultado\n"+ (primeiroNumero-segundoNumero));
        }else if (escolha == 3){
        System.out.println("Resultado\n"+ (primeiroNumero*segundoNumero));
        }else if (escolha == 4){
         System.out.println("Resultado\n"+ (primeiroNumero/segundoNumero));
        }else{
        System.out.println("opcão inválida");
        }
    
}
    }